src/machine/state/mem.rs

```diff
@@ -2,10 +2,10 @@ use machine::state::types::*;
 
 use std::ops::{Index, IndexMut};
 
-pub struct ByteAt(u64);
-pub struct WydeAt(u64);
-pub struct TetraAt(u64);
-pub struct OctaAt(u64);
+pub struct ByteAt(pub u64);
```

Comment:
Please do not change the interface of the memory in a pull-request that implements the instructions. If you want to change an interface, please start a new pull-request or issue for this "problem". Making this change here has the ability to cause enormous trouble for anyone who needs to work with the memory, e.g. the one who implements it.

Btw: These fields are private for a good reason: The From and Into-Traits abstract over the actual implementation.